UPGRADE-6.1.md

```diff
@@ -0,0 +1,8 @@
+UPGRADE FROM 6.0 to 6.1
+=======================
+
+Security
+--------
+
+* Deprecate `RememberMeHandler` not implementing `getUserIdentifierForCookie()`; `AbstractRememberMeHandler` has a default implementation
```

Comment:
Missing spaces before the stars

Reply:
And here I thought I had added those spaces when I committed this... they're added now.